app/scripts/components/datasets/s-explore/index.tsx

```diff
@@ -61,6 +64,25 @@ import {
   BASEMAP_ID_DEFAULT
 } from '$components/common/mapbox/map-options/basemaps';
 
+const BackLink = styled.a`
```

Comment:
```suggestion
const BackLink = styled(Link)`
```

Reply:
(You would need a line of importing Link component from react-router-dom around line 10 
`import { Link } from 'react-router-dom';`